<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="ARPGL000810" ignored="2" name="Test" tests="15" failures="3" timestamp="2020-09-01T16:49:38 IST" time="320.724" errors="0">
  <testcase name="loginPageTitleTest" time="0.015" classname="com.qa.testcases.LoginPageTest">
    <skipped/>
  </testcase> <!-- loginPageTitleTest -->
  <testcase name="UITest" time="0.173" classname="com.qa.testcases.LoginPageTest"/>
  <testcase name="stdUserLoginTest" time="60.294" classname="com.qa.testcases.LoginPageTest">
    <skipped/>
  </testcase> <!-- stdUserLoginTest -->
  <testcase name="stdUserLoginTest" time="9.428" classname="com.qa.testcases.LoginPageTest"/>
  <testcase name="multiUserLoginTest" time="9.655" classname="com.qa.testcases.LoginPageTest"/>
  <testcase name="multiUserLoginTest" time="0.183" classname="com.qa.testcases.LoginPageTest">
    <skipped/>
  </testcase> <!-- multiUserLoginTest -->
  <testcase name="multiUserLoginTest" time="0.185" classname="com.qa.testcases.LoginPageTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.qa.pages.ProductsPage.verifyPageByURL(ProductsPage.java:35)
at com.qa.testcases.LoginPageTest.multiUserLoginTest(LoginPageTest.java:65)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- multiUserLoginTest -->
  <testcase name="multiUserLoginTest" time="3.775" classname="com.qa.testcases.LoginPageTest"/>
  <testcase name="multiUserLoginTest" time="9.757" classname="com.qa.testcases.LoginPageTest"/>
  <testcase name="sortingTest" time="5.934" classname="com.qa.testcases.ProductsPageTest"/>
  <testcase name="sortingTest" time="7.426" classname="com.qa.testcases.ProductsPageTest">
    <skipped/>
  </testcase> <!-- sortingTest -->
  <testcase name="sortingTest" time="4.194" classname="com.qa.testcases.ProductsPageTest">
    <failure type="java.lang.AssertionError" message="expected [Sauce Labs OnesieM] but found [Sauce Labs Onesie]">
      <![CDATA[java.lang.AssertionError: expected [Sauce Labs OnesieM] but found [Sauce Labs Onesie]
at com.qa.testcases.ProductsPageTest.sortingTest(ProductsPageTest.java:53)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- sortingTest -->
  <testcase name="sortingTest" time="38.853" classname="com.qa.testcases.ProductsPageTest"/>
  <testcase name="sortingTest" time="15.269" classname="com.qa.testcases.ProductsPageTest">
    <skipped/>
  </testcase> <!-- sortingTest -->
  <testcase name="sortingTest" time="11.283" classname="com.qa.testcases.ProductsPageTest">
    <failure type="java.lang.AssertionError" message="expected [Sauce Labs BackpackM] but found [Sauce Labs Backpack]">
      <![CDATA[java.lang.AssertionError: expected [Sauce Labs BackpackM] but found [Sauce Labs Backpack]
at com.qa.testcases.ProductsPageTest.sortingTest(ProductsPageTest.java:53)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- sortingTest -->
  <testcase name="@AfterSuite extentReportCleanUp" time="0.0" classname="com.qa.base.TestBase">
    <ignored/>
  </testcase> <!-- @AfterSuite extentReportCleanUp -->
  <testcase name="@BeforeSuite extentReportSetup" time="0.0" classname="com.qa.base.TestBase">
    <ignored/>
  </testcase> <!-- @BeforeSuite extentReportSetup -->
</testsuite> <!-- Test -->
